drivers/dai/intel/dmic/dmic.c

```diff
@@ -566,19 +566,14 @@ static void dai_dmic_start(struct dai_intel_dmic *dmic)
 
 	dai_dmic_start_fifo_packers(dmic, dmic->dai_config_params.dai_index);
 
-	for (i = 0; i < CONFIG_DAI_DMIC_HW_CONTROLLERS; i++) {
 #ifdef CONFIG_SOC_SERIES_INTEL_ADSP_ACE
-		dai_dmic_update_bits(dmic, dmic_base[i] + CIC_CONTROL,
-				     CIC_CONTROL_SOFT_RESET, 0);
-
-		LOG_INF("dmic_start(), cic 0x%08x",
-			dai_dmic_read(dmic, dmic_base[i] + CIC_CONTROL));
+	for (i = 0; i < CONFIG_DAI_DMIC_HW_CONTROLLERS; i++)
+		dai_dmic_update_bits(dmic, dmic_base[i] + CIC_CONTROL, CIC_CONTROL_SOFT_RESET, 0);
```

Comment:
put { } around the loop.

Reply:
Is it coding style or a technical thing?